src/pudl/package_data/nrelatb/column_maps/data.csv

```diff
@@ -0,0 +1,21 @@
+year_index,2019,2020,2021,2022,2023
+report_year,atb_year,atb_year,atb_year,atb_year,atb_year
+core_metric_key,core_metric_key,core_metric_key,core_metric_key,core_metric_key,core_metric_key
+core_metric_parameter,core_metric_parameter,core_metric_parameter,core_metric_parameter,core_metric_parameter,core_metric_parameter
+core_metric_case,core_metric_case,core_metric_case,core_metric_case,core_metric_case,core_metric_case
+cost_recovery_period_years,crpyears,crpyears,crpyears,crpyears,crpyears
+technology_description,technology,technology,technology,technology,technology
+technology_alias,,,technology_alias,technology_alias,technology_alias
+display_name,,,display_name,display_name,display_name
+is_default,,,default,default,default
```

Comment:
i couldn't find documentation about what this field actually is. There are only ever nulls 0's and 1's.... it looks like a quasi boolean but i can't tell.

Reply:
Blocking: can you add the docs you're referencing here to `pudl.docs.data_sources.nrelatb`?

Reply:
I was about to go add em but then got turned around because the docs i went digging into are a [series of web pages](https://atb.nrel.gov/electricity/2023/definitions). these docs are mostly pdfs with some html of the empty forms. Does that still seem reasonable? and if so should I just extract the html from the few docs pages per year that I've been pawing through? this feels a little off brand from the rest of these docs.

Reply:
Hm... this is the thing we would automatically link in our ReadtheDocs page for that data source, and I see some value in having an archived link of the dataset documentation in case it all gets taken down, particularly for older datasets. As long as the file name reflects what it contains (e.g., 2023 ATB Definitions, 2023 ATB Scenarios) I think archiving them should make sense so long as they aren't prohibitively large. I would imagine these pages aren't changing more than once per year. But it's not a hill I'll die on, if you don't think this makes sense don't worry about it.

Reply:
hm well i think that actually the info on the read the docs page is just linked to the info in `metadata.sources`. i believe the docs pdfs/html form are just for internal use. I could be wrong about that though.